Limber/MIXMockServer.cpp

```diff
@@ -67,6 +73,35 @@ class SimpleMIXChannelRegistry {
         std::unordered_set<std::string> channels_;
 };
 
+class MIXParticipantInformation {
```

Comment:
Regarding MIXParticipantInformation class, the data members are optional to indicate whether the field has been set or not. Instead of using two different maps for roster and proxyJID, I am saving them as one MIXParticipantInformation class. But both roster and proxyJID are set differently, roster is set when client->requestRoster is called and proxyJID is set when channel is successfully joined.